src/CSSFramework/BootstrapCSSFramework.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace TheWebmen\ElementalGrid\CSSFramework;
+
+use DNADesign\Elemental\Models\BaseElement;
+
+final class BootstrapCSSFramework implements CSSFrameworkInterface
+{
+    /**
```

Comment:
Do we want to support PHP7.3 and lower?

If yes; The dockblocks looks fine.
If not; Why not just typehint variables/returns instead of using docblocks?

Reply:
Yes, as for example our client (OM) is still at 7.2. Type hinting will be added in the future. It already is built in the articles module